README.md

```diff
@@ -16,14 +58,66 @@ docker run \
 -v /var/run/docker.sock:/var/run/docker.sock \
 -v D:\host\path\where\to\output\singularity\image:/output \
 --privileged -t --rm \
-singularityware/docker2singularity \
+singularityware/docker2singularity:2.4 \
 ubuntu:14.04
 ```
 
 Replace `D:\host\path\where\to\output\singularity\image` with a path on the host filesystem where your Singularity image will be created. Replace `ubuntu:14.04` with the docker image name you wish to convert (it will be pulled from Docker Hub if it does not exist on your host system).
 
 `docker2singularity` uses the Docker daemon located on the host system. It will access the Docker image cache from the host system avoiding having to redownload images that are already present locally.
 
+## Build a Squashfs Image
```

Comment:
Could you mention here what is the minimal version of Singularity required to run Squashfs images?

Reply:
I can find when support was added in the source code, but I don't recall if there were any issues. If they were they would have been fixed later. I will cite when support was added and if users have issues I will help them.